### PR TITLE
RGP: DDP-6523: Fixed code using proxy for SendGrid for Seqr export co…

### PR DESCRIPTION
**Pull request by charlottedye**
…de email to look for proxy in correct configuration file and improve logging

## Context

This is a fix for DDP-6523.  That code accidentally got the proxy URL from the wrong configuration file

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java [hunk 1]
```diff
@@ -111,12 +113,14 @@ private void sendNotificationEmail(boolean isFailure, boolean isSkip) {
         } else {
             subject = cfg.getString(CustomExportConfigFile.EMAIL_SUCCESS_SUBJECT);
             templateId = cfg.getString(CustomExportConfigFile.EMAIL_SUCCESS_TEMPLATE_ID);
-            String fileUrl = "https://console.cloud.google.com/storage/browser/" + exportCfg.getString(CustomExportConfigFile.BUCKET_NAME)
-                    + "/" + exportCfg.getString(CustomExportConfigFile.FILE_PATH);
+            String fileUrl = "https://console.cloud.google.com/storage/browser/" + mainCfg.getString(CustomExportConfigFile.BUCKET_NAME)
+                    + "/" + mainCfg.getString(CustomExportConfigFile.FILE_PATH);
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java]: `mainCfg` is not the `CustomExportConfigFile`, is it?

**Review by yufengwng — CHANGES_REQUESTED** [review]

File: pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java (hunk 1 above)
Comment (charlottedye) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java]: Nope.  It comes from here: ` public static void register(Scheduler scheduler, Config cfg) throws SchedulerException {`

Comment (charlottedye) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java]: Wait let me check this one more time.  I swear I did this right but now I'm worried

Comment (charlottedye) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java]: Nope; wrote a totally different bug.  Clearly I'm a little off today!

**Review by yufengwng — APPROVED** [review]

File: pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java (hunk 1 above)
Comment (yufengwng) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java]: Looks much better!

**Pull request merged**